webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1656 @@
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "documentLifecycle": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "106"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "frameId": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "84"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15"
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "frameType": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "84"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "initiator": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "84"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15"
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "method": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "84"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15"
```

Comment:
`onRuleMatchedDebug` and related are not supported by Safari.

Reply:
```suggestion
                    "version_added": false
```